### PR TITLE
Enhancement: Add `rector/rector`

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,6 +25,10 @@ static-code-analysis-baseline: check-symfony vendor ## Generates a baseline for
 tests: check-symfony vendor
 	symfony php vendor/bin/phpunit tests
 
-.PHONY: vendor
+.PHONY: vendorr
```
Comment (OskarStark) [inline Makefile:28]:
```suggestion
.PHONY: vendor
```

**Comment by OskarStark** [comment]
Ready to merge from my side, please make the refactoring CI job required after merging, thanks

**Comment by slunak** [comment]
This or release first?

**Comment by OskarStark** [comment]
Merge first if possible

**Pull request merged**